src/middleware/router.js

```diff
@@ -215,16 +215,17 @@ function sendRequestToRoutes (ctx, routes, next) {
                 ctx.autoRetry = true
                 ctx.error = responseObj.error
               }
-
               // then set koa response from responseObj.response
-              return setKoaResponse(ctx, responseObj.response)
+              /* return */ setKoaResponse(ctx, responseObj.response)
             } else {
-              return setKoaResponse(ctx, response)
+              /* return */ setKoaResponse(ctx, response)
```

Comment:
Lets maybe remove the commented out `return`, unless this is just commented out temporarily for now and needs to be added back in later on.